src/software/ai/evaluation/find_open_areas.cpp

```diff
@@ -1,26 +1,33 @@
 #include "software/ai/evaluation/find_open_areas.h"
 
+#include <optional>
+
 #include "proto/parameters.pb.h"
 #include "software/geom/algorithms/find_open_circles.h"
 
-std::vector<Circle> findGoodChipTargets(const World& world)
+std::vector<Circle> findGoodChipTargets(const World& world,
+                                        std::optional<Rectangle> target_area_rectangle)
 {
     double inset     = 0.3;  // Determined experimentally to be a reasonable value
     double ballX     = world.ball().position().x();
     double fieldX    = world.field().enemyGoalCenter().x() - inset;
     double negFieldY = world.field().enemyCornerNeg().y() + inset;
     double posFieldY = world.field().enemyCornerPos().y() - inset;
```

Comment:
this isn't related to your changes but by convention, we use ball_x, field_x, neg_field_y and pos_field_y